canonical-websites/www.ubuntu.com
# Add the beta whitepaper

## Done
Add the robotics beta whitepaper

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to /whitepapers/robotics and check it all works
- Go to /whitepapers/beta/robotics and check it all works

Super-seeds: https://github.com/canonical-websites/www.ubuntu.com/pull/4847/
